indico/modules/events/editing/operations.py

```diff
@@ -283,3 +289,47 @@ def update_review_condition(condition, file_types):
 def delete_review_condition(condition):
     logger.info('Review condition %r deleted by %r', condition, session.user)
     db.session.delete(condition)
+
+
+def assign_editor(editable, editor):
+    editable.editor = editor
+    db.session.flush()
+    logger.info('Editable %r assigned to editor %r', editable, editor)
```

Comment:
Shouldn't this also be logged to the event log? Might be useful for event managers to see who performed an assignment and when...